aspnet/fundamentals/logging.rst

```diff
@@ -126,105 +180,251 @@ Debug          dbug
  * `Serilog <https://github.com/serilog/serilog-framework-logging>`_ - provider for the Serilog library
 
+Some third-party frameworks can do `semantic logging, also known as structured logging <http://programmers.stackexchange.com/questions/312197/benefits-of-structured-logging-vs-basic-logging>`__.  
+
+To use a third-party framework, the process is typically similar to what you've seen for built-in providers: add the appropriate NuGet package to your project and call an extension method on ``ILoggerFactory``. For more information, see each framework's documentation.  
+
 You can create your own custom providers as well, to support other logging frameworks or your own internal logging requirements.
 
 Logging Recommendations
 -----------------------
 
-The following are some recommendations you may find helpful when implementing logging in your ASP.NET Core applications.
+Here are some recommendations you may find helpful when implementing logging in your ASP.NET Core applications.
 
-1. Log using the correct ``LogLevel``. This will allow you to consume and route logging output appropriately based on the importance of the messages.
+1. Log using the appropriate ``LogLevel``. This will allow you to consume and route logging output appropriately based on the importance of the messages.
 
 2. Log information that will enable errors to be identified quickly. Avoid logging irrelevant or redundant information.
 
 3. Keep log messages concise without sacrificing important information.
 
-4. Although loggers will not log if disabled, consider adding code guards around logging methods to prevent extra method calls and log message setup overhead, especially within loops and performance critical methods.
+4. Although loggers will not log if disabled, consider adding code guards around logging methods to prevent extra method calls and log message setup overhead, especially within loops and performance-critical methods.
 
-5. Name your loggers with a distinct prefix so they can easily be filtered or disabled. Remember the ``Create<T>`` extension will create loggers named with the full name of the class.
+5. Name your loggers with a distinct prefix so they can easily be filtered or disabled. Use the ``CreateLogger<T>`` extension to create loggers named with the full name of the class.
 
 6. Use Scopes sparingly, and only for actions with a bounded start and end. For example, the framework provides a scope around MVC actions. Avoid nesting many scopes within one another.
 
-7. Application logging code should be related to the business concerns of the application. Increase the logging verbosity to reveal additional framework-related concerns, rather than implementing yourself.
+7. Application logging code should be related to the business concerns of the application. Increase the logging verbosity to reveal additional framework-related logs, rather than calling logger methods yourself for framework events.
 
 Summary
```

Comment:
ARG - Summaries are banned unless you make a compelling case - which you have not.